buildbots.rst

```diff
@@ -105,7 +105,7 @@ introducing additional failures should generally not be an option.
 Flags-dependent failures
 ------------------------
 
-Sometimes, while you have run the :doc:`whole test suite <runtests>` before
+Sometimes, while you have run the :doc:`whole test suite <testing/index>` before
```

Comment:
```suggestion
Sometimes, while you have run the :ref:`whole test suite <runtests>` before
```

Isn't it better to link the `:ref:` rather than the `:doc:`, like you did elsewhere, to avoid things breaking if either this document or the testing one is ever moved or renamed again? At the very least, its worth using an absolute rather than relative path so if and when this document gets moved into a subdir, the link won't break (really only makes sense to use relative paths within the same subdirectory tree).